Wazebar.js

```diff
@@ -572,10 +572,17 @@
   }
 
   function BuildCurrentStateEntries() {
+    const topCountry = wmeSDK.DataModel.Countries.getTopCountry();
```

Comment:
This fails when called on a forum page because wmeSDK is not used on a forum page.

Reply:
Thank you.  Updateing the function to accout for `if (!forumPage)` berfore making the SDK call.  Thanks for the catch.

```suggestion
    function BuildCurrentStateEntries() {
    var currentState = "";

    if (!forumPage) {
      const topCountry = wmeSDK.DataModel.Countries.getTopCountry();
      const topCountryId = topCountry ? topCountry.id : null;

      if (topCountryId === 235) {
        // Only proceed if the top country is the US
        var currState = getCurrentState();
        if (!currState || !States[currState]) {
          return currentState; // Return an empty string if currState or its corresponding States entry is invalid.
        }
        currentState += '<div class="WazeBarText WazeBarCurrState" id="' + currState.replace(" ", "_") + 'ForumCurrState"><a href="' + States[currState].forum.replace("https://www.waze.com", location.origin) + '" ' + LoadNewTab() + ">" + States[currState].abbr + "</a></div>";
        currentState += '<div class="WazeBarText WazeBarCurrState"><a href="' + States[currState].wiki + '" target="_blank">' + States[currState].abbr + " Wiki</a></div>";
      }
    }
    return currentState;
  }
```